.typos.toml

```diff
@@ -21,6 +21,7 @@ Setted = "Setted"
 hve = "hve"
 
 [type.md.extend-words]
-# This is used as "yoh-key" in markdown files to describe the pronounciation 
+# This is used as "yoh-key" in markdown files to describe the pronunciation 
 # for Youki
 "yoh" = "yoh"
+"typ" = "typ"
```

Comment:
I think this needs to go in the rust.extended-words section, because this is in code not md files. The CI still fails https://github.com/youki-dev/youki/actions/runs/14998226105/job/42183137249?pr=3168

Reply:
I have pushed a commit fixing this.